inc/classes/class-imagify-admin-ajax-post.php

```diff
@@ -27,8 +27,8 @@ class Imagify_Admin_Ajax_Post extends Imagify_Admin_Ajax_Post_Deprecated {
 		'imagify_manual_optimize',
 		'imagify_manual_reoptimize',
 		'imagify_optimize_missing_sizes',
-		'imagify_generate_webp_versions',
-		'imagify_delete_webp_versions',
+		'imagify_generate_nextgen_versions',
```

Comment:
:warning: This function has already been refactored here: 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-admin-ajax-post.php#L30